core/templates/dev/head/combined-tests.spec.ts

```diff
@@ -46,12 +46,14 @@ getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting()
 );
-// Then we find all the tests. Note that known failing files are exempted;
+// Then we find all the tests and controller, directive, service/factory files
```

Comment:
Then we find all the test files, as well as any controller, directive, service/factory files.

Reply:
Done